src/overlays/actors/ovl_Bg_Bdan_Objects/z_bg_bdan_objects.c

```diff
@@ -378,50 +398,50 @@ void func_8086C874(BgBdanObjects* this, PlayState* play) {
     }
 }
 
-void func_8086C9A8(BgBdanObjects* this, PlayState* play) {
+void BgBdanObjects_WaitForSwitch(BgBdanObjects* this, PlayState* play) {
     if (Flags_GetSwitch(play, this->switchFlag)) {
         this->timer = 100;
-        this->actionFunc = func_8086C9F0;
+        this->actionFunc = BgBdanObjects_ChangeWaterboxLevel;
     }
 }
 
-void func_8086C9F0(BgBdanObjects* this, PlayState* play) {
+void BgBdanObjects_ChangeWaterboxLevel(BgBdanObjects* this, PlayState* play) {
```

Comment:
"water box" so
```suggestion
void BgBdanObjects_ChangeWaterBoxLevel(BgBdanObjects* this, PlayState* play) {
```
<img src="https://cdn.discordapp.com/emojis/943102210366119956.webp?size=56&quality=lossless" alt="skawoUHHUH" height="24">